assets/translations/es/messages.private.json

```diff
@@ -0,0 +1,64 @@
+{
+  "BotGreeting": "¿Como puedo ayudar?",
+  "ChatInputUserNotMemberDisabledText": "No eres participante de este chat",
+  "ConfirmEndEngagementConfirmButton": "Si, terminé",
+  "ConfirmEndEngagementDenyButton": "No, regresar",
+  "ConfirmEndEngagementMessage": "Finalizar este chat?",
+  "Connecting": "Conectando â€¦",
+  "DialpadAsteriskCaption": "*",
+  "DialpadAsteriskDescription": "",
+  "DialpadHashCaption": "#",
+  "DialpadHashDescription": "",
+  "DialpadNum0Caption": "0",
+  "DialpadNum0Description": "+",
+  "DialpadNum1Caption": "1",
+  "DialpadNum1Description": "â€”",
```

Comment:
Yeah, this was printed from the strings object and I left them because didn't knew if they were on purpose.. I'll remove those, but what should be here instead? It keeps printing the same character.

Reply:
A quick google search turns up that [this is the long hyphen or em-dash](https://markmcb.com/2011/11/07/replacing-with-utf-8-characters-in-ruby-on-rails/).